src/controllers/trainee/routes.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express';
+import trainee from './Controller';
+
+const router: Router = Router();
+
+router.get('/', trainee.get);
+router.post('/', trainee.post);
+router.put('/:id', trainee.put);
```

Comment:
pass id in request body instead of params for put route.

Reply:
changes done